base/commands/map/map_it_test.go

```diff
@@ -146,3 +149,49 @@ func size_InteractiveTest(t *testing.T) {
 		})
 	})
 }
+
+func destroy_NonInteractiveTest(t *testing.T) {
+	it.MapTester(t, func(tcx it.TestContext, m *hz.Map) {
+		t := tcx.T
+		ctx := context.Background()
+		tcx.WithReset(func() {
+			check.Must(m.Set(ctx, "foo", "bar"))
+			require.Equal(t, 1, check.MustValue(m.Size(ctx)))
+			go tcx.WriteStdin([]byte("y\n"))
+			check.Must(tcx.CLC().Execute(ctx, "map", "-n", m.Name(), "destroy"))
+			require.Equal(t, 0, check.MustValue(m.Size(ctx)))
```

Comment:
Instead of checking the size of the map, it would be better to check its existence. You can use ` tcx.Client.GetDistributedObjectsInfo()` to get the list of objects in the cluster and check whether `m.Name()` is in the list.